editor/views/editoritem.py

```diff
@@ -511,10 +511,51 @@ def get(self, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         ei = self.get_object()
+        error_message = ''
```

Comment:
Since this code is specific to exams, it shouldn't be in the generic EditorItem publish view.
We need a new PublishView in `views/exam.py`, which has `model = NewExam`. And since there are only two kinds of EditorItem, we might as well make a PublishView in `views/question.py` as well, and get rid of this one.